app/frontend/src/components/admin/inspection/LocationEdit.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <v-dialog v-model="dialog" max-width="800px">
+    <template v-slot:activator="{ on }">
+      <!-- <v-btn color="primary" dark v-on="on">Open Dialog</v-btn> -->
+      <v-btn v-if="true" v-on="on" color="primary" class="my-3" small>Edit Submission</v-btn>
+    </template>
+    <v-card>
+      <v-form ref="form" v-model="updateValid">
+        <v-card-title>
+          <span class="headline">
+            Editing submission from:
+            <strong>{{businessName}}</strong>
+          </span>
+        </v-card-title>
+        <v-card-text>
+          <v-container class="pa-0">
+
+            <!-- child Location form component -->
+            <Location :reviewMode="reviewMode"/>
+
+          </v-container>
+        </v-card-text>
+        <v-card-actions class="pa-3 pb-10">
+          <v-btn color="primary" @click="update">Save</v-btn>
+          <v-btn text @click="cancelUpdate">Cancel</v-btn>
+        </v-card-actions>
+      </v-form>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+import Vuex from 'vuex';
+import { mapGetters } from 'vuex';
+
+import Location from '@/components/form/Location.vue';
+import { SilvipcRoles } from '@/utils/constants';
+
+/* eslint-disable */
+import ipcService from '../../../services/ipcService';
+
+const localStore = new Vuex.Store({
```

Comment:
so.. is this where I create a 'local store'?
i'm not sure why i would lazy load anything if i have a store for this component

Reply:
You don't want to be defining a store at all here. When you lazy load a vuex module, the framework will bring that into memory.